GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
```

Comment:
Given the new chapter, shall we remove "Q: How do I craft?"? Is the content of the answer contained in the new chapter?

Reply:
I think it would be reasonable. Crafting is an important part of the game. Thus, it deserves to be in the main text. But it is just a suggestion. What do you think?